### DISCUSSION TITLE
Pricing / license

### DISCUSSION THREAD

**Discussion by jeromebaum** [digitallyinduced/ihp#1360] [Q&A] [answered]
The `LICENSE` file says MIT. There's also a "pricing" page on your website that offers different licenses for a fee.

Could you explain how all of this works? Are there parts of IHP that are not open-source?

**Comment by mpscholten** [comment, marked as answer]
Hey, you can find some more info on the announcement page here https://ihp.digitallyinduced.com/blog/6392ad84-e96a-46ce-9ab5-1d9523d48b23-announcing-the-new-ihp-developer-subscription-ihp-v0-14 and here https://ihp.digitallyinduced.com/Guide/ihp-pro.html

The majority of IHP is open source and available in this repo. Few extensions (like the stripe integration which you can use to make money with your IHP project) are closed source and only available for people that subscribe to IHP Pro.

Let me know if you have further questions :)

**Reply by jeromebaum** [reply]
Thanks, this is helpful. It might be worth linking this from the pricing page for easy discovery.
